sections/03_categorize.md

```diff
@@ -218,17 +222,36 @@ datasets are quite small, even for simple machine learning algorithms. The
 challenge is greater for deep learning models with many parameters. While
 unsupervised and semi-supervised approaches can help with small sample sizes,
 the field would benefit greatly from large collections of anonymized records in
-which a substantial number of records have undergone expert review.
-
-This challenge is not unique to EHR-based studies. Work on medical images,
--omics data in applications for which detailed metadata are required, and other
+which a substantial number of records have undergone expert review. This
+challenge is not unique to EHR-based studies. Work on medical images, -omics
+data in applications for which detailed metadata are required, and other
 applications for which labels are costly to obtain will be hampered as long as
-abundant curated data are unavailable. Unsupervised and semi-supervised methods
-provide one path forward [@doi:10.1016/j.jbi.2016.10.007], as do adversarial
-training examples [@doi:10.1101/095786]. We also expect the recently described
-data programming strategy [@arxiv:1605.07723] to play a role in addressing this
-challenge. In data programming, noisy automated labeling functions are
-integrated. Numerous writers have described data as the new oil
+abundant curated data are unavailable.
+
+Successful approaches to date in this domain have sidestepped this challenge by
+making methodological choices that either reduce the need for labeled examples
+or that use transformations to training data to increase the number of times it
+can be used before overfitting occurs. For example, the unsupervised and
+semi-supervised methods that we've discussed reduce the need for labeled
+examples [@doi:10.1016/j.jbi.2016.10.007]. The adversarial training example
+strategies that we've mentioned can reduce overfitting, if transformations are
+available that preserve the meaningful content of the data while transforming
+irrelevant features [@doi:10.1101/095786]. While adversarial training examples
+can be easily imagined for certain methods that operate on images, it's more
+challenging to figure out what an equivalent transformation would be for a
+patient's clinical test results. Consequently, it may be hard to employ
+adversarial training examples, not to be confused with adversarial neural
```

Comment:
Specifically, not to be confused with "generative adversarial neural networks" or is it meant to be more broad?  I'm wondering whether the GAN terminology will help readers connect this to other places we discuss them.